mne/bem.py

```diff
@@ -1556,6 +1560,31 @@ def write_bem_surfaces(fname, surfs, overwrite=False, verbose=None):
             end_file(fid)
 
 
+@verbose
+def write_head_bem(fname, rr, tris, on_defects='raise', overwrite=False,
+                   verbose=None):
+    """Write a head surface to a fiff file.
+
+    Parameters
+    ----------
+    fname : str
+        Filename to write.
+    rr : array, shape=(n_vertices, 3)
+        Coordinate points.
```

Comment:
```suggestion
        Coordinate points in the head coordinate system.
```

Reply:
In this case I think it should be the MRI coordinate system (part of the `bem/*` files)

Reply:
Yes, it's the MRI coordinate system.